python/graphstorm/run/gsgnn_np/gsgnn_np.py

```diff
@@ -85,10 +81,17 @@ def main(config_args):
         trainer.restore_model(model_path=config.restore_model_path,
                               model_layer_to_load=config.restore_model_layers)
     trainer.setup_device(device=get_device())
+    train_idxs = train_data.get_node_train_set(config.target_ntype, mask="train_mask")
```

Comment:
As by default the `get_node_train_set` uses "train_mask" as mask name, we can remove the mask argument here to simplify our own code.